hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -176,6 +179,11 @@ def move_output_files(output, reference, prod_dir, long_output, include_los_disp
         outName = "{}_inc_map.tif".format(os.path.join(prod_dir, long_output))
         shutil.copy(inName, outName)
 
+    if water_masking:
+        inName = "final_mask.tif"
```

Comment:
I think I'd prefer this to be `final_water_mask.tif` or something with "water" in it.

Reply:
changed to "final_water_mask.tif", and also made change in water_mask.py